src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -347,6 +358,37 @@ public boolean isActive()
     }
 
     /**
+     * Gets a boolean value indicating whether or not this instance is
+     * streaming.
+     *
+     * @param performTimeoutCheck  when true, it requires fresh data and not
+     * just the active property to be set.
+     *
+     * @return true if this instance is streaming, false otherwise.
+     */
+    public boolean isActive(boolean performTimeoutCheck)
+    {
+        if (active && performTimeoutCheck)
+        {
+            if (lastReceivedFrame == null)
+            {
+                return false;
+            }
+            else
+            {
+                long silentIntervalMs
```

Comment:
'silent' here is referring to no video packets received?  might be a bit confusing with 'audio silence'.  `timeSinceLastReceivedFrameMs` maybe?